src/View/Service/index.html.twig

```diff
@@ -13,11 +13,19 @@
         <h2 class="service-title pl-lg-3">Nos services</h2>
         <div class="row justify-content-center">
             {% for service in services %}
-            <div class="col-md-6 card-deck">
-                <div class="card service-card">
+            <div class="col-md-6 card-deck card-order">
+                <div class="card service-card mb-4">
                     <img src="/assets/images/{{ service.image }}" class="card-img-top" alt="{{ service.name }}">
                     <div class="card-body card-description">
-                        <h3 class="card-title">{{ service.name }}</h3>
+                        <h3 class="card-title text-center
+                        {% if loop.first or loop.last %}
```

Comment:
non tu ne t'y prends pas bien
si tu veux faire un sur deux, c'est plutot if loop.index0 is even ou is odd
mais ce qui serait encore mieux, ce serait de gérer ça uniquement coté CSS. Là tu n'aurait qu'une class card
et en CSS avec :nth-child(event) et :nth-child(odd) tu pourrait modifier juste ta couleur

Reply:
justement ce n'est pas une sur deux pour éviter d'avoir une colonne bleu et une colonne rose.
Mais en effet, je vais essayer de gérer ça avec le css